src/UserGuide/V1.2.x/User-Manual/Tiered-Storage_timecho.md

```diff
@@ -19,6 +19,78 @@
 
 -->
 
-# Tiered Storage
+# Tiered Storage 
+## Overview
 
-TODO
\ No newline at end of file
+Tiered storage function provides users with the ability to manage tiered storage media. users can use the tiered storage function to configure different types of storage media for IoTDB and to classify the storage media. ioTDB can support tiered storage from memory, SSD, normal hard disc to network hard disc by parameter configuration only according to the degree of hot and cold data. Specifically, in IoTDB, the configuration of tiered storage is reflected in the management of multiple directories. Users can group tiered storage directories into the same category and configure them into IoTDB as a "tier", which is called storage tier; at the same time, users can categorize data according to hot or cold, and store different categories of data into designated storage tiers. Meanwhile, users can categorise data according to hot or cold and store different categories of data in the specified tier. Currently, IoTDB supports the classification of hot and cold data by TTL, when the data in one tier does not meet the TTL rules defined in the current tier, the data will be automatically migrated to the next tier.
```

Comment:
The Tiered storage functionality allows users to define multiple layers of storage, spanning across multiple types of storage media (Memory mapped directory, SSD, rotational hard discs or cloud storage). While memory and cloud storage is usually singular, the local file system storages can consist of multiple directories joined together into one tier. In order to decide where to store data, the probably most important configuration option is the TTL (Time to live / age) of data.   Therefore for each tier a default TTL needs to be provided. Each tier also has a quota to how much data it can contain. When inserting data, IoTDB checks if the conditions for the first layer match (Is the data too old? Is the layer already saturated with too much data?) If any of the conditions don't match, the next tier is checked until a matching tier is found.